src/pages/Search/SearchFilters.tsx

```diff
@@ -58,8 +80,8 @@ function SearchFilters({query}: SearchFiltersProps) {
                         interactive
                         title={item.title}
                         icon={item.icon}
-                        iconWidth={variables.iconSizeLarge}
-                        iconHeight={variables.iconSizeLarge}
+                        iconWidth={variables.iconSizeNormal}
+                        iconHeight={variables.iconSizeNormal}
```

Comment:
Why this change?

Reply:
Because of [this comment](https://github.com/Expensify/App/pull/42184#issuecomment-2127216129)